datasource/src/main/scala/org/locationtech/rasterframes/datasource/raster/RasterSourceRelation.scala

```diff
@@ -40,13 +41,16 @@ import org.locationtech.rasterframes.tiles.ProjectedRasterTile
   * @param catalogTable Specification of raster path sources
   * @param bandIndexes band indexes to fetch
   * @param subtileDims how big to tile/subdivide rasters info
+  * @param lazyTiles if true, creates a lazy representation of tile instead of fetching contents.
+  * @param spatialIndexPartitions if not None, adds a spatial index. If Option value < 1, uses the value of `numShufflePartitions` in SparkContext.
```

Comment:
if the value is >=1 does it become the number of partitions used? that is not exactly clear from a quick read. could epxand this